WebRoot/public/launchZCS.jsp

```diff
@@ -58,6 +58,17 @@
 	// Prevent IE from ever going into compatibility/quirks mode.
 	response.setHeader("X-UA-Compatible", "IE=edge");
 %>
+<%
+    // check if help directory exists
+    boolean productHelpSupported = true;
+	try {
+		productHelpSupported = new java.io.File(application.getRealPath("/help")).exists();
+	} catch (Exception ignored) {
+		// Just in case there's anException, catch it.
```

Comment:
```suggestion
		// Just in case there's an Exception, catch it.
```
or 
```suggestion
		// Just in case there's an exception, catch it.
```

Reply:
Updated the comment, please review @k-kato .